src/components/Card/Card.css

```diff
@@ -1,31 +1,69 @@
-a {
-  text-decoration: none;
+.card-container {
+  position: relative;
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-direction: row;
```

Comment:
I might not remember correctly, but flex-direction default value is row so we don't need to specify it?

Reply:
Whoops - I always forget about that!